contracts/switchboard/native/PolygonL1Switchboard.sol

```diff
@@ -21,7 +21,7 @@ contract PolygonL1Switchboard is NativeSwitchboardBase, FxBaseRootTunnel {
      * @notice This modifier overrides the onlyRemoteSwitchboard modifier in the NativeSwitchboardBase contract
      */
     modifier onlyRemoteSwitchboard() override {
-        require(true, "ONLY_FX_CHILD");
+        revert("ONLY_FX_CHILD");
```

Comment:
Redundant check here. This will revert in all cases. Submitted a bug report https://gist.github.com/sujithsomraaj/cd2512b8cd04721e650d1cc2823283c0

Reply:
Intended to revert since packet is received via different mechanism from polygon bridge